freqtrade/optimize/optimize_reports/bt_output.py

```diff
@@ -370,6 +370,7 @@ def text_table_add_metrics(strat_results: dict) -> None:
                 f"{strat_results['winning_days']} / "
                 f"{strat_results['draw_days']} / {strat_results['losing_days']}",
             ),
+            ("Max trade duration", f"{strat_results['holding_max']}"),
```

Comment:
Using it this way will result in a keyError when running `backtesting-show` on an existing result. 
you'll need to use .get with a reasonable defualt (probably N/A or similar).